invoicing_app/management/commands/generate_tax_receipts.py

```diff
@@ -166,24 +167,14 @@ def handle(self, **options):
 
         if options['event_id']:
             self.event_id = options['event_id']
-            self.query = self.query.replace(
-                'CONDITION_MASK',
-                'AND `Events`.`id` = ' + str(self.event_id)
-            )
+            self.conditional_mask = 'AND `Events`.`id` = {}'.format(self.event_id)
 
         if options['user_id']:
             self.user_id = options['user_id']
-            self.query = self.query.replace(
-                'CONDITION_MASK',
-                'AND `Events`.`uid` = ' + str(self.user_id)
-            )
+            self.conditional_mask = 'AND `Events`.`uid` = {}'.format(self.user_id)
 
-        # If there isn't condition, remove the CONDITION MASK from the query
-        if (not options['user_id']) and (not options['event_id']):
-            self.query = self.query.replace(
-                'CONDITION_MASK',
-                ''
-            )
+        # The self.conditional_mask is put into the query
+        self.query = self.query.format(condition_mask=self.conditional_mask, parent_child_mask='{parent_child_mask}')
```

Comment:
I think using format only to change the first parameter and leave the second parameter doesn't look good. I think it is better to apply the format when you have both `condition_maks` and `parent_child_mask` values.

Reply:
I think you're right. Fixed!